src/drafts/index.ts

```diff
@@ -7,3 +7,15 @@
 export * from '../NavList'
 export * from '../Dialog/Dialog'
 export * from '../SegmentedControl'
+
+export {default as InlineAutocomplete} from '../InlineAutocomplete'
+export type {
+  InlineAutocompleteProps,
+  ShowSuggestionsEvent,
+  Suggestion,
+  Suggestions,
+  Trigger
+} from '../InlineAutocomplete'
+
+export {default as MarkdownViewer} from '../MarkdownViewer'
+export type {MarkdownViewerProps, InteractiveMarkdownViewerProps} from '../MarkdownViewer'
```

Comment:
This is missing the exports to access the MarkdownEditor

```suggestion
export type {MarkdownViewerProps, InteractiveMarkdownViewerProps} from '../MarkdownViewer'
export {default as MarkdownEditor} from '../MarkdownEditor'
export type {MarkdownEditorProps} from '../MarkdownEditor'
```